cmd/zetaclientd/start.go

```diff
@@ -173,14 +172,15 @@ func start(_ *cobra.Command, _ []string) error {
 
 	// Wait for TSS keygen to be successful before proceeding, This is a blocking thread only for a new keygen.
 	// For existing keygen, this should directly proceed to the next step
-	ticker := time.NewTicker(time.Second * 1)
-	for range ticker.C {
-		if cfg.Keygen.Status != observerTypes.KeygenStatus_KeyGenSuccess {
-			startLogger.Info().Msgf("Waiting for TSS Keygen to be a success, current status %s", cfg.Keygen.Status)
-			continue
-		}
-		break
-	}
+	// Temporarily allow zeta-client to proceed even if the keygen is set to pending
```

Comment:
Is it going to be put back for `v12.1.0`?

Reply:
yes